options_test.go

```diff
@@ -238,3 +238,52 @@ func TestWithHostPortAccess(t *testing.T) {
 		})
 	}
 }
+
+func TestWithImageMount(t *testing.T) {
```

Comment:
question: does this need a docker version check which calls https://pkg.go.dev/testing#T.Skip if its not high enough?

Reply:
Done in [a143f15](https://github.com/testcontainers/testcontainers-go/pull/3044/commits/a143f15bf34d0b3e0d8c31562799cb655f02695f)